client/app/bundles/shared/components/buttonApp.jsx

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+
+class buttonApp extends React.Component {
+   render() {
+       
+       var buttonStyle = {
+         fontSize: 18,
+         color: '#6D0839',
+         fontWeight: "bold",
+         backgroundColor: '#FFFFFF',
+         paddingTop: 17,
+         paddingBottom: 16,
+         paddingLeft: 15,
+         paddingRight: 15,
+         width: 203,
+         borderRadius: 5,
+         borderColor: '#FFFFFF'
+      }
+      return (
+          <div>
+          <ButtonFilledM/>
+          <ButtonGhostM/>
+          <ButtonFilled/>
+          <ButtonGhost/>
+          <ButtonGhostXS/>
+          </div>
+      );
+   }
+}
+class ButtonFilledM extends React.Component {
+   render() {
+       
+       var buttonStyle = {
+         fontSize: 18,
+         color: '#6D0839',
```

Comment:
Let's create some kind of constants.jsx file with these values and use similar names as `app/assets/stylesheets/base/_values.scss` and `app/assets/stylesheets/base/_colors.scss`.